src/python/WMCore/Database/Dialects.py

```diff
@@ -12,6 +13,3 @@
     from sqlalchemy.dialects.mysql.base import MySQLDialect
     from sqlalchemy.dialects.sqlite.base import SQLiteDialect
     from sqlalchemy.dialects.oracle.base import OracleDialect
-
-
-__all__ = [MySQLDialect, SQLiteDialect, OracleDialect]
```

Comment:
From what I read, it would only have a different behavior if we were import it like `import *`, so it looks good.

Reply:
Yes.